EIPS/eip-4361.md

```diff
@@ -0,0 +1,313 @@
+    - EIP-191 produces more human-readable messages, while EIP-712 creates signing outputs for machine consumption, with most wallets not displaying the payload to be signed in a manner friendly to humans. **(1)**![](https://i.imgur.com/yvhz2mD.png)
+
+    - EIP-712 has the advantage of on-chain representation and on-chain verifiability, such as for their use in metatransactions, but this feature is not relevant for the specification's scope. **(2)**
+- Why not use JWTs? Wallets don't support JWTs. The keccak hash function is [not assigned by IANA](https://www.iana.org/assignments/jose/jose.xhtml) for use as a JOSE algorithm. **(2, 3)**
+- Why not use YAML or YAML with exceptions? YAML is loose compared to ABNF, which can readily express character set limiting, required ordering, and strict whitespacing. **(2, 3)**
+
+### Discussions and Acknowledgements
+- @awoie and @rh7 led the discussion on EIP-712 and EIP-191 signing schemes, including the benefits of structured data types and potential for extensions when using EIP-712.
+- @awoie surfaced a number of privacy and security considerations related to MITM attacks, replay attacks, relay attacks, and RP-binding.
+- @pedrouid suggested a YAML-inspired message format that struck the balance between human and machine readable.
+- @Amxx suggested adding the `version` indicator to allow forwards-compatibility, and also brought up important implications between EIP-1271 and network selection.
+- @mhluongo led discussion around i18n support and suggesting several improvements around the message format.
+- @oed suggested low-impact changes that could do a lot to enable capabilities-based permissioning models.
+- @kdenhartog advised on mitigating/calling out phishing attacks, domain binding security, OIDC models of identity, and examples of using EIP-712 to structure data.
+- @SamWilsn made several improvements to the ABNF grammar and additional security guidelines to wallet implementers to prevent phishing attacks.
+- @cfelde suggested a minor improvement to the ABNF grammar.
+
+### Out of Scope
+The following concerns are out of scope for this version of the specification to define:
+- Additional authentication not based on Ethereum addresses.
+- Authorization to server resources.
+- Interpretation of the URIs in the `resources` term as claims or other resources.
+- The specific mechanisms to ensure domain-binding.
+- The specific mechanisms to generate nonces and evaluation of their appropriateness.
+- Protocols for use without TLS connections.
+
+## Backwards Compatibility
+- Most wallet implementations already support EIP-191, so this is used as a base pattern with additional features.
+- Requirements were gathered from existing implementations of similar sign-in workflows, including statements to allow the user to accept a Terms of Service, nonces for replay protection, and inclusion of the Ethereum address itself in the message.
+
+## Forwards Compatibility
```

Comment:
```suggestion
## Forwards Compatibility
```
This is not a valid EIP section. This sort of information should be incorporated into the rationale.

Reply:
![Screenshot 2021-12-06 at 16-19-50 Meme Generator - Imgflip](https://user-images.githubusercontent.com/57262657/144924359-bc467640-3814-4932-96bf-68ed0607f225.png)